backend/app/utils.py

```diff
@@ -273,6 +265,11 @@ def appstream2dict(appstream_url=None) -> dict[str, dict]:
         if app["is_free_license"] == "":
             app["is_free_license"] = False
 
+        # The new appstream spec uses a new <developer> key, so backfill the old
+        # field for backwards compatibility
+        if developer_name := app.get("developer", {}).get("name"):
```

Comment:
I think `developer` can be a list too? So this might fail in those cases?

Reply:
Hmm, possibly. Can fix that later.